package.json

```
@@ -74,8 +74,8 @@
     "@hmcts/frontend": "0.0.39-alpha",
     "@hmcts/media-viewer": "2.7.9",
     "@hmcts/nodejs-healthcheck": "^1.7.0",
+    "@hmcts/rpx-xui-common-lib": "1.4.25-search-location-selection-getfocus-fontsize",
     "@hmcts/properties-volume": "^0.0.13",
```

```suggestion
    "@hmcts/properties-volume": "^0.0.13",
    "@hmcts/rpx-xui-common-lib": "1.4.25-search-location-selection-getfocus-fontsize",
```
If this can be alphabetical order like before